frontend/src/core/cells/cell.ts

```diff
@@ -65,18 +66,9 @@ export function transitionCell(
       // The cell didn't run, but it was intentional, so don't count as
       // errored.
       nextCell.stopped = true;
-      // Update elapsed time, since it is still useful to know how long before
-      // the cell was stopped.
-      nextCell.runElapsedTimeMs = cell.runStartTimestamp
-        ? (message.timestamp - cell.runStartTimestamp) * 1000
-        : null;
     } else {
+      // Communicate that the cell errored (e.g., an exception was raised)
       nextCell.errored = true;
-      // Update elapsed time, since it is still useful to know how long before
-      // the cell was stopped.
-      nextCell.runElapsedTimeMs = cell.runStartTimestamp
-        ? (message.timestamp - cell.runStartTimestamp) * 1000
-        : null;
```

Comment:
@mscolnick this was the fix, this is already calculated in the switch statement on message status, when an `idle` message is sent